src/components/Card.js

```diff
@@ -46,11 +66,13 @@ export default class Card {
   getView() {
     // get card view
     this._card = this.getTemplate();
+    this._likeButton = this._card.querySelector(".gallery__card-button");
     this._card.querySelector(".gallery__card-image").src = this._link;
```

Comment:
`this._card.querySelector(".gallery__card-image")`

**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.

```js
  getView() {
    this._card = this.getTemplate();
    this._likeButton = this._card.querySelector(".gallery__card-button");
    this._cardImage = this._card.querySelector(".gallery__card-image")
    this._trashButton = this._card.querySelector(".gallery__card-delete")

```